### PR TITLE
Various fixes to the orgs branch

### PR DESCRIPTION
**Pull request by blomquisg**
Update settings.yaml.example
Use unscoped query in installation media db migration
Fix create_domains permission
Fix select_organization default data in loader
Force users in no org to see no organizationally bound objects

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -31,10 +31,8 @@ def self.apply_org_scope scope
     if SETTINGS[:orgs_enabled] and not User.current.admin?
       org_ids = Organization.current
       org_ids = [org_ids] unless org_ids.is_a? Array
-      if org_ids.any?
-        org_ids = org_ids.map(&:id)
-        scope = scope.joins(:organizations).where("organizations.id in (?)", org_ids)
-      end
+      org_ids = org_ids.any? ? org_ids = org_ids.map(&:id) : [""]
```
Comment (ohadlevy) [inline app/models/organization.rb]: can be simplified to:

  org_ids.map!(&:id)

File: config/settings.yaml.example [hunk 2]
```diff
@@ -6,3 +6,5 @@
 :puppetconfdir: /etc/puppet/puppet.conf
 :login: false
 :require_ssl: false
+:single_org: false
```
Comment (ohadlevy) [inline config/settings.yaml.example]: can you add a oneliner telling the users whats the diff between the org options?

Comment (blomquisg) [inline config/settings.yaml.example]: Yep, will do.

On Thu, Jul 26, 2012 at 2:36 PM, Ohad Levy <
reply@reply.github.com

File: app/models/organization.rb (hunk 1 above)
Comment (blomquisg) [inline app/models/organization.rb]: Cool.   Gotta love ruby :)

I'll test this out and change it accordingly.

On Thu, Jul 26, 2012 at 2:35 PM, Ohad Levy <
reply@reply.github.com

**Comment by blomquisg** [comment]
Closing with fixes in subsequent pull request

**Pull request closed without merging**